### ISSUE TITLE
[NL - Amsterdam] - Backbase B.V. - Senior iOS Engineer (Strategy and Propositions)

### ISSUE THREAD

**Issue by arthurpalves** [CocoaHeadsBrasil/vagas#774]
## Backbase
Backbase is disrupting the banking industry. Worldwide, renowned banks and financial institutions build their businesses using our mobile apps. What we do impacts the banking experience of millions of users every single day.

Whether or not you’ve heard of us, you and your friends have probably used some of the products we’ve created. Backbase accelerates the transformation of digital banking. From being a Dutch startup company five years ago, we are now an international market leader. Despite having a leading market position, we haven’t lost our startup mentality and we’re not done innovating just yet. Working in teams with highly talented and skilled professionals from all over the world, we share a drive to make things better, nicer, and more advanced. More than 400 specialists dedicate themselves to what we do best, which is transforming global financial services with inspiring, innovative solutions. At Backbase, we love to create outstanding experiences, not just for our customers and users, but for ourselves, and this is what motivates us to transform our industry by delivering our best work.

## Position
Senior iOS Engineer (Strategy and Propositions).

## Job Type
Permanent.

## As a Senior iOS Engineer on Strategy and Propositions...

**TL;DR:** You don't adopt innovation, you drive it! 🚘😎You help create the trends; the present is not enough for you.

You will deliver jaw-dropping, award-worthy iOS experiences. You will push the envelope when implementing modern UI/UX patterns, across the board, including motion design and animations. Last but definitely not least: you will always go the extra mile to create a pixel-perfect experience. From a personal perspective, you are well-rounded and enjoy working with multiple cultures and stakeholders. When it comes to the crunch, you love the pressure of a healthy deadline.

Your role is to keep our innovation power running at the fastest pace possible and help us remain at the cutting edge. Are you ready to innovate?

You are passionate about delivering great experiences. You know everything about modern UI/UX-patterns and love polishing the look and feel of your applications;

In our teams, we use the following technologies:
* Swift / Python / Ruby
* Java / Kotlin
* The latest trends

You are familiar with clean coding, the SOLID principles and know your design patterns and how to apply them in real life. You are aware of the importance of a well-designed architecture. You are not afraid of proposing new designs and get into discussions with your peers about new ways of solving problems. We like to keep work interesting and we encourage our developers to develop their personal strengths. Whether it’s deep-diving into technology, Agile processes, or moving into full-stack development, at Backbase we offer you the room to grow.

###### Required
* Fluency in English.
* Undoubtedly, your iOS development experience and knowledge are second to none;
* You have top-notch, hands-on development skills and always create clean, maintainable and reusable product-quality code;
* You are comfortable in a Scrum team, using Lean and Agile methodologies. You have experience in TDD, unit testing, continuous integration or Jenkins;
* You should be able to boast of a few apps that you have successfully developed yourself that WOW us;
* You can communicate your thoughts clearly to developers, designers and business stakeholders;
* You are a fast learner, capable of getting in-depth knowledge on the product and the company within a short amount of time;
* You’ve seen or worked with different cultures, giving you the ability to easily adapt to the Backbase cultural mix;
* You share our “DNA”: self-starting – entrepreneurial – resilient and outspoken;
* You must be a self-starter, able to work well under pressure and have a keen eye for detail;
* Having a ‘can do’ mentality is of more importance than many years of experience; you really know how to get stuff done;
* You basically think and act like an entrepreneur; experience in running your own business is a big plus;
* You love to over deliver, without under promising;
* We won’t have to tell you anything about software architectures or web technologies; your expert knowledge is up to date;
* Nice to have: a natural presenter or willing to learn how to deliver a jaw-dropping presentation.

## Benefits
* Market rate salary
We work hard, play hard, and earn hard

* Visa sponsorship & relocation package
For you, your partner, children and pets

* Multicultural environment
Work and play with people from around the world

* Contributory pension scheme
We’ll help you fill your coffers for old age

* Location
We work in beautiful Amsterdam. Nuff said, right?

* 25 days annual leave
Our full-timers know how to cut loose and recharge

* Company fitness plan
Burn off your free lunch and reenergise!

* Social events
Beach, boat, beers and board games. We do the lot!

* Free lunch & refreshments
Stay healthy and focused with food, drinks and good company

## Who you will be working with
You will work with hundreds of people from all around the world, from 60+ nationalities, but with 32+ Brazilians 🇧🇷 in our Amsterdam office alone you won't get rid of Portuguese so easily.

You will be a key member in an Agile scrum team composed of a Product Owner, a Senior Interaction Designer and multiple top-notch developers. They’re already looking forward to meeting you

Take a look at a small bit of our life here: https://www.instagram.com/life_at_backbase

## Contact me ;)
arthura@backbase.com
